app/components/CreateCrashRecordModal.tsx

```diff
@@ -0,0 +1,377 @@
+    }>({ crash });
+
+    if (responseData && responseData.insert_crashes_cris) {
+    onSubmitCallback();
+    }
+    reset();
+  };
+
+  return (
+    <Modal
+      show={show}
+      onHide={() => {
+        reset();
+        onClose();
+      }}
+      animation={false}
+      backdrop="static"
+    >
+      <Modal.Header closeButton>
+        <Modal.Title>Create crash record</Modal.Title>
+      </Modal.Header>
+      <Modal.Body>
+        <Form onSubmit={handleSubmit(onSubmit)} id="userForm">
+            {/* Case Id */}
+          <Form.Group className="mb-3">
+            <Form.Label>Case ID</Form.Label>
+            <Form.Control
+              {...register("case_id", { required: true })}
+              autoComplete="off"
+              data-1p-ignore
+              placeholder="Case ID"
+              isInvalid={Boolean(errors.case_id)}
+              autoFocus
+            />
+            <Form.Control.Feedback>Required</Form.Control.Feedback>
+            <Form.Control.Feedback type="invalid">
+              Case ID is required
+            </Form.Control.Feedback>
+          </Form.Group>
+          {/* Crash date */}
+          <Form.Group className="mb-3">
+            <Form.Label>Crash date</Form.Label>
+            <FormControlDatePicker<CrashInputs>
+              name="crash_timestamp"
+              control={control}
+              isInvalid={Boolean(errors?.crash_timestamp)}
+              {...{ rules: { required: true } }}
+            />
+            <Form.Control.Feedback
+              type="invalid"
+              style={{
+                // manually control date picker feedback
+                display: Boolean(errors?.crash_timestamp) ? "block" : "none",
+              }}
+            >
+              Crash date is required
+            </Form.Control.Feedback>
+          </Form.Group>
+          {/* Primary address */}
+          <Form.Group className="mb-3">
+            <Form.Label>Primary address</Form.Label>
+            <Form.Control
+              {...register("rpt_street_name", {
+                required: true,
+                setValueAs: (v) => v?.trim() || null,
+              })}
+              autoComplete="off"
+              data-1p-ignore
+              placeholder="ex: W 4TH ST"
+              isInvalid={Boolean(errors.rpt_street_name)}
+            />
+            <Form.Control.Feedback type="invalid">
+              Primary address is required
+            </Form.Control.Feedback>
+          </Form.Group>
+          {/* Secondary address */}
+          <Form.Group className="mb-3">
+            <Form.Label>Secondary address</Form.Label>
+            <Form.Control
+              {...register("rpt_sec_street_name", {
+                setValueAs: (v) => v?.trim() || null,
+              })}
+              autoComplete="off"
+              data-1p-ignore
+              placeholder="ex: 100 S CONGRESS AVE"
+              isInvalid={Boolean(errors.rpt_street_name)}
```

Comment:
@johnclary i saw red highlighting on this when I tested and had an invalid primary address, and I think this just need to be updated to `rpt_sec_street_name`.